ansible/roles/debops-contrib.kodi/tasks/main.yml

```diff
@@ -30,3 +30,18 @@
     state: '{{ "present" if (kodi__deploy_state == "present") else "absent" }}'
     system: True
 # ]]]
+
+- name: Create polkit configuration
+  template:
+    src: 'etc/polkit-1/localauthority/50-local.d/kodi-actions.pkla.j2'
+    dest: '/etc/polkit-1/localauthority/50-local.d/kodi-actions.pkla'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
+  when: (kodi__polkit_action != "")
+
+- name: Remove polkit configuration
+  file:
+    path: '/etc/polkit-1/localauthority/50-local.d/kodi-actions.pkla'
+    state: 'absent'
+  when: (kodi__polkit_action == "")
```

Comment:
You could try to replace that with:
```yaml
when: not kodi__polkit_action|d()
```
It should have the same result.

Reply:
Thanks for the hint. Fixed.